Make sure downloadsFolder isnt translated

was previously in patch-toolkit_components_downloads_DownloadUIHelper_jsm
Index: toolkit/components/downloads/DownloadIntegration.sys.mjs
--- toolkit/components/downloads/DownloadIntegration.sys.mjs.orig
+++ toolkit/components/downloads/DownloadIntegration.sys.mjs
@@ -883,7 +883,7 @@ export var DownloadIntegration = {
     // displayed in the user interface.
     let directoryPath = PathUtils.join(
       this._getDirectory(aName),
-      lazy.stringBundle.GetStringFromName("downloadsFolder")
+      "Downloads"
     );
 
     // Create the Downloads folder and ignore if it already exists.
